<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>619</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>550</width>
      <height>619</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/images/backgroundimage.jpg);
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>550</width>
      <height>619</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(18, 36, 33);
</string>
    </property>
    <widget class="QLineEdit" name="usernameCred">
     <property name="geometry">
      <rect>
       <x>137</x>
       <y>350</y>
       <width>275</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(208, 253, 246, 0.94);
color:rgb(3, 22, 19);
border-radius: 4px;
padding-left: 10px;
padding-bottom: 3px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter Username Here</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordCred">
     <property name="geometry">
      <rect>
       <x>137</x>
       <y>400</y>
       <width>275</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(208, 253, 246, 0.94);
color:rgb(3, 22, 19);
border-radius: 4px;
padding-left: 10px;
padding-bottom: 3px;
</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Enter Password Here</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>137</x>
       <y>450</y>
       <width>275</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#loginButton{
background-color:rgb(119, 205, 249);
color:rgb(3, 22, 19);
border-radius:5px;
}

QPushButton#loginButton:pressed{
background-color:rgb(208, 253, 246);
}</string>
     </property>
     <property name="text">
      <string>L o g i n </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>70</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/shovelandpickaxesymbol.png);
background-color: rgb(18, 36, 33);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/images/wizardhatsymbol.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>351</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI Variable Text Semibold</family>
       <pointsize>45</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Trail Wizards</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>260</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Dig. Build. Shred.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>520</y>
       <width>491</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
</string>
     </property>
     <property name="text">
      <string>PSST! Hey you! Yeah you. First time here? No worries! Register here now and become a Trail Wizard!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="toRegisterPageButton">
     <property name="geometry">
      <rect>
       <x>232</x>
       <y>560</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#toRegisterPageButton{
background-color:rgb(119, 205, 249);
color:rgb(3, 22, 19);
border-radius:5px;
}

QPushButton#toRegisterPageButton:pressed{
background-color:rgb(208, 253, 246);
}</string>
     </property>
     <property name="text">
      <string>Register</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
